templates/distribution/manifests/dr/patches/velero-schedule-full.yml.tpl

```
@@ -10,3 +10,7 @@ metadata:
   namespace: kube-system
 spec:
   schedule: {{ .spec.distribution.modules.dr.velero.schedules.cron.full }}
+  template:
+    {{- if ne .spec.distribution.modules.dr.velero.schedules.ttl "" }}
+    ttl: {{ .spec.distribution.modules.dr.velero.schedules.ttl }}
+    {{- end }}
```

With this logic we are setting the defaults value to `""` to say "not set" and inheriting the default value from the module that will be hard to debug, imagine that you check the default values file and you see `""` but the actual value in the cluster you see is `720h` (as in the field description).

If would be better to use `index` or similar to check if the value is defined by the user:

```suggestion
    {{- if index .spec.distribution.modules.dr.velero.schedules "ttl" }}
    ttl: {{ .spec.distribution.modules.dr.velero.schedules.ttl }}
    {{- end }}
```

and drop the default value from the defaults file. Same thing for the cron expressions.